SYCL/Basic/get_backend.cpp

```diff
@@ -1,4 +1,4 @@
-// UNSUPPORTED: TEMPORARY_DISABLED
+// REQUIRES: TEMPORARY_DISABLED
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: env SYCL_DEVICE_FILTER=%sycl_be %t.out
```

Comment:
@vladimirlaz, either the comment below or this RUN command must be updated - they contradict each other.

// Do not set SYCL_DEVICE_FILTER. We do not want the preferred
// backend.